.cirrus.yml

```
@@ -179,26 +209,44 @@ analyze_task:
       NODE_VERSION: 22
   depends_on:
     - build
+  <<: *MAVEN_CACHE
+  <<: *RUNTIME_CACHE
+  <<: *NPM_CACHE
+  <<: *NPMRC_SCRIPT_DEFINITION
+  <<: *JAVA_COVERAGE_CACHE
+  test_java_script:
+    - source cirrus-env QA
+    - mvn install -T1C -Pcoverage,coverage-report
+
+analyze_task:
+  eks_container:
+    <<: *CONTAINER_DEFINITION
+    dockerfile: .cirrus/nodejs.Dockerfile
+    docker_arguments:
+      CIRRUS_AWS_ACCOUNT: ${CIRRUS_AWS_ACCOUNT}
+      NODE_VERSION: 22
+  depends_on:
+    - test_js
+    - test_java
   env:
     # analysis on next
     SONAR_TOKEN: VAULT[development/kv/data/next data.token]
     SONAR_HOST_URL: https://next.sonarqube.com/sonarqube
-    ARTIFACTORY_DEPLOY_ACCESS_TOKEN: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-qa-deployer access_token]
   <<: *MAVEN_CACHE
   <<: *RUNTIME_CACHE
   <<: *NPM_CACHE
   <<: *NPMRC_SCRIPT_DEFINITION
-  build_and_analyze_script:
-    - source cirrus-env QA
-    - npm run bridge:build:cov
-    - PULL_REQUEST_SHA=$GIT_SHA1 regular_mvn_build_deploy_analyze -P-deploy-sonarsource,-release,-sign -Dcommercial -Dmaven.shade.skip=true -Dmaven.install.skip=true -Dmaven.deploy.skip=true -Dsonar.coverage.jacoco.xmlReportPaths=$CIRRUS_WORKING_DIR/report-aggregate/target/site/jacoco-aggregate/jacoco.xml
+  <<: *JS_COVERAGE_CACHE
+  <<: *JAVA_COVERAGE_CACHE
+  analyze_script:
+    - source cirrus-env BUILD
+    - PULL_REQUEST_SHA=$GIT_SHA1 regular_mvn_build_deploy_analyze -P-deploy-sonarsource,-release,-sign -Dcommercial -Dmaven.shade.skip=true -Dmaven.install.skip=true -Dmaven.deploy.skip=true -DskipTests
```

I believe we might not need the `-DskipTests` option here as it is redundant with `-Dmaven.install.skip=true`. WDYT?

```suggestion
    - PULL_REQUEST_SHA=$GIT_SHA1 regular_mvn_build_deploy_analyze -P-deploy-sonarsource,-release,-sign -Dcommercial -Dmaven.shade.skip=true -Dmaven.install.skip=true -Dmaven.deploy.skip=true
```

---

Install is a latter point in the mvn lifecycle. It is different from the test step. https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html